recipes/openscenegraph-osgQt/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "openscenegraph-osgQt" %}
+{% set version = "3.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openscenegraph/osgQt/archive/{{ version }}.tar.gz
+  sha256: dcc4436590639168e6470fe9c5343c82bca738d3296ebee014f40f2dc029afa1
+  patches:
+    - patches/osx.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
```

As this primarily provides a shared library 
```suggestion
  skip: true  # [win]
  run_exports:
    - {{ pin_subpackage(name, max_pin='x.x') }}
```

To check if the ABI tends to be stable between major, minor and patch versions [ABI Laboratory](https://abi-laboratory.pro/index.php?view=tracker) is one of the best places. You can also look/ask upstream to see if they have any policies around this. If you can't find anything a good approach is:
* If new version releases are very rare, pinning tightly is appropriate `max_pin='x.x.x.x.x.x.x.x'`
* If versions are more frequent, pinning on minor version releases is often okay

For more details see [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements).

---

Done on update mata.yml commit